crates/api/src/local_user/notifications/unread_count.rs

```diff
@@ -11,7 +11,8 @@ pub async fn unread_count(
 ) -> LemmyResult<Json<GetUnreadCountResponse>> {
   let person_id = local_user_view.person.id;
 
-  let replies = CommentReplyView::get_unread_replies(&mut context.pool(), person_id).await?;
+  let replies =
+    CommentReplyView::get_unread_replies(&mut context.pool(), &local_user_view.local_user).await?;
 
   let mentions = PersonMentionView::get_unread_mentions(&mut context.pool(), person_id).await?;
```

Comment:
This same logic needs to get added to `PersonMentionView` also. 

IE, pass in the `local_user` to `PersonMentionView::get_unread_mentions` just like you're doing for the CommentReplyView.

You probably don't need to add unit tests for that one, unless you'd like to.

Reply:
added, rust tests added as well as i had to touch them anyway to create the local user.